test/concurrent/BlockedQueueTest.ooc

```diff
@@ -14,16 +14,16 @@ import threading/Thread
 BlockedQueueTest: class extends Fixture {
 	init: func {
 		super("BlockedQueue")
-		version (!windows) { this add("cover", This _testWithCover) }
+		this add("cover", This _testWithCover)
 		this add("class", This _testWithClass)
 	}
 	_testWithCover: static func {
 		queue := BlockedQueue<Int> new()
-		numberOfThreads := 8
+		numberOfThreads := 4
```

Comment:
This never failed in the same way AFAIK, I just changed it to 4 threads to be "consistent" and avoid problems similar to those in the sync resource test. I still don't really know why that test failed but not this one. OK to stick to four for now and then evaluate if we can change all the threading tests later (if we need to)?